main.js

```diff
@@ -363,19 +359,6 @@ ipcMain.on('window-showChildWindow', (event, windowOptions) => {
     childWindow.focus();
   }
 
-
-  // show the child window when it will be ready
-  new Promise(resolve => {
-    if (childWindowIsReadyToShow) {
-      resolve();
-      return;
-    }
-    ipcMain.once('window-childWindowIsReadyToShow', () => resolve());
-  }).then(() => {
-    // The child window will show itself when rendered
-    childWindow.send('window-setContents', windowOptions);
-  });
```

Comment:
We're able to remove this now because of the new system of updating the child window contents by setting it in the main window and then waiting for it to get synced to the child window.